src/game/behaviors/fish.inc.c

```diff
@@ -1,163 +1,299 @@
-// fish.c.inc
+/**
+ * @file fish.inc.c
+ * Implements behaviour and spawning for fish located in the Secret Aquarium and other levels.
+ */
 
-void ActionFish0(void) {
+/**
+ * Spawns fish with settings chosen by the field o->oBehParams2ndByte.
+ * These settings are animations, colour, and spawn quanitity.
+ * Fish spawning restricted to a set distance from Mario.
+ */
+void fish_act_spawn(void) {
     s32 i;
-    s32 sp38;
+    s32 schoolQuantity;
     s16 model;
-    f32 sp30;
-    struct Animation **sp2C;
-    struct Object *sp28;
+    f32 minDistToMario;
+    struct Animation **fishAnimation;
+    struct Object *fishObject;
     switch (o->oBehParams2ndByte) {
+        
+        // Blue fish with a quanitiy of twenty.
```

Comment:
Typo. I'm probably not catching all typos so it might be worth running a spellcheck